app/src/main/res/common/values/dimens.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="spacing_extreme_large">40dp</dimen>
+    <dimen name="spacing_extreme_super_large">50dp</dimen>
+    <dimen name="spacing_super_large">32dp</dimen>
+    <dimen name="spacing_large">24dp</dimen>
+    <dimen name="spacing_medium">16dp</dimen>
+    <dimen name="spacing_small">8dp</dimen>
+    <dimen name="spacing_small_half">4dp</dimen>
+    <dimen name="spacing_smallest">2dp</dimen>
+</resources>
```

Comment:
Why do we need this many spacings, if the only ones we're using are `spacing_medium` and `spacing_small`?

Reply:
Yes man! my bad 😞